hash/hash.go

```diff
@@ -5,26 +5,8 @@ import (
 	"github.com/NethermindEth/starknet.go/contracts"
 	"github.com/NethermindEth/starknet.go/curve"
 	"github.com/NethermindEth/starknet.go/rpc"
-	"github.com/NethermindEth/starknet.go/utils"
 )
 
-// ComputeHashOnElementsFelt computes the hash on elements of a Felt array.
-//
-// Parameters:
-// - feltArr: A pointer to an array of Felt objects.
-// Returns:
-// - *felt.Felt: a pointer to a Felt object
-// - error: an error if any
-func ComputeHashOnElementsFelt(feltArr []*felt.Felt) (*felt.Felt, error) {
```

Comment:
Moved to the `curve` package because of golang "import cycle" with `contracts` pkg. Also, original ComputeHashOnElements is there too